ext/http/00_serve.js

```diff
@@ -632,37 +634,56 @@ function serve(arg1, arg2) {
     listenOpts.cert = options.cert;
     listenOpts.key = options.key;
     listenOpts.alpnProtocols = ["h2", "http/1.1"];
-    const listener = Deno.listenTls(listenOpts);
+    listener = Deno.listenTls(listenOpts);
     listenOpts.port = listener.addr.port;
-    context.initialize(op_http_serve(
-      listener.rid,
-    ));
   } else {
-    const listener = Deno.listen(listenOpts);
+    listener = Deno.listen(listenOpts);
```

Comment:
Not directly related to this PR, but why are these two using `Deno.listen` and `Deno.listenTls` instead of using direct import from other ext?

Reply:
In theory there's no reason why we don't use it. I can make that change while I'm here.

Reply:
Thanks!